contracts/p1/StRSR.sol

```diff
@@ -335,6 +335,54 @@ abstract contract StRSRP1 is Initializable, ComponentP1, IStRSR, EIP712Upgradeab
         IERC20Upgradeable(address(rsr)).safeTransfer(account, rsrAmount);
     }
 
+    function cancelUnstake(uint256 endId) external notPausedOrFrozen {
+        address account = _msgSender();
+
+        // == Refresh ==
+        assetRegistry.refresh();
+
+        // == Checks + Effects ==
+        require(basketHandler.fullyCollateralized(), "RToken uncollateralized");
+        require(basketHandler.status() == CollateralStatus.SOUND, "basket defaulted");
```

Comment:
@tbrent I'm contemplating removing these two checks given that we're not removing RSR from the system. What do you think?

Reply:
Absolutely. You _should_ be able to re-stake when undercollateralized, or when basket is not sound.

Reply:
We can also remove the assetRegistry refresh, I see no reason to need that

Reply:
Yeah I think you're right about the refresh. I only kept it if under collateralized was a factor. Removed it.